src/App.vue

```diff
@@ -53,9 +53,34 @@ export default {
     },
 
     style() {
-      return this.$route.name === 'game-board' && this.wallpaperUrl
-        ? `background-image: url('${this.wallpaperUrl}')`
-        : null;
+      return {
+        'background-image':
+          this.$route.name === 'game-board' &&
+          this.wallpaperUrl
+            ? `url('${this.wallpaperUrl}')`
+            : null,
+        '--border-radius':
+          this.settings &&
+          this.platform &&
+          this.settings[this.platform.code] &&
+          !this.settings[this.platform.code].borderRadius
+            ? '0px'
+            : null,
+        '--list-background':
+          this.settings &&
+          this.platform &&
+          this.settings[this.platform.code] &&
+          this.settings[this.platform.code].hideListBackgrounds
+            ? 'transparent'
+            : null,
```

Comment:
instead of having a setting to hideListBackground can we set the background in the theme as transparent?